client/verta/verta/_repository/commit.py

```diff
@@ -62,28 +64,37 @@ def _lazy_load_blobs(self):
 
         self._loaded_from_remote = True
 
-    def __repr__(self):
+    def show(self):
```

Comment:
`show` is not the right name (it's supposed to only show diffs), but I haven't been able to come up with a better name yet. Essentially, I'm not sure we always want to display the full contents of the commit. I was thinking mainly about `log`. Maybe the default should be to display everything and the log uses something to fetch just the header part without the contents. TBD once we have log.

Reply:
Changed to `describe`.